system-int-test/test/cmr/system_int_test/search/keyword_endpoint_test.clj

```diff
@@ -697,181 +697,99 @@
+                  "uuid" "b26761fa-8d8e-4bd8-a8ba-db6575554ad7"}
+                 {"value" "application/zip",
+                  "uuid" "4e5db77b-bc1d-4f7c-9f13-e3e54e0b2e3b"}
+                 {"value" "application/vnd.google-earth.kmz",
+                  "uuid" "f7328bf5-8ef2-4f95-a4e0-6fb16d122237"}
+                 {"value" "application/vnd.ms-excel",
+                  "uuid" "7c99ff72-5239-424d-a0bf-9712c33ea76d"}
+                 {"value" "application/pdf",
+                  "uuid" "627269ae-ba93-492e-8c31-cc4de1d69810"}
+                 {"value" "application/x-hdfeos",
+                  "uuid" "b1eac265-2b00-4c39-a429-797c13a2c640"}
+                 {"value" "image/png", "uuid" "edb9e800-ec31-4d5c-848d-c548fd151db2"}
+                 {"value" "text/xml", "uuid" "091b6afc-ab75-4790-8e71-3b32ae8bd0a4"}
+                 {"value" "image/gif", "uuid" "ad61b259-8131-4e0e-aac8-a800a0a51ca6"}
+                 {"value" "application/x-bufr",
+                  "uuid" "e384b8a8-8cec-4230-9ebe-4db76bbef706"}
+                 {"value" "application/x-vnd.iso.19139-2+xml",
+                  "uuid" "c1a8dbb7-312d-4481-998e-58d126b32080"}
+                 {"value" "image/bmp", "uuid" "b7687b8f-7a24-4150-bd9d-28e0d53f7554"}
+                 {"value" "application/vnd.google-earth.kml+xml",
+                  "uuid" "80045dcb-18ee-463a-8baf-ffcabed510ea"}
+                 {"value" "application/x-hdf",
+                  "uuid" "b0a3e733-4d1b-486f-b56c-c405a5e4367b"}
+                 {"value" "text/javascript",
+                  "uuid" "40cb0bdd-67f4-43c8-aa57-2bdc260e6950"}
+                 {"value" "image/vnd.collada+xml",
+                  "uuid" "d3ef6fe7-b6cd-45b4-9a27-d42fa3289116"}
+                 {"value" "application/x-tar-gz",
+                  "uuid" "5e70beda-396e-4cc8-bdd5-70dfc8a1142e"}
+                 {"value" "application/opensearchdescription+xml",
+                  "uuid" "07bcc60e-1551-44d9-b87e-7c260d230ecb"}
+                 {"value" "application/xml",
+                  "uuid" "dd6c5cea-4100-4973-9ba9-659fdd7fd608"}
+                 {"value" "application/x-netcdf",
+                  "uuid" "2b192915-32a8-4b68-a720-8ca8a84f04ca"}
+                 {"value" "text/markdown",
+                  "uuid" "b403039f-a107-4a84-88a1-29e4d1b30b0b"}
+                 {"value" "text/html", "uuid" "415a10b5-7286-4195-a88e-00c7b995b7d0"}
+                 {"value" "application/octet-stream",
+                  "uuid" "b77e64ef-ce80-4dab-b552-c6062990a6e0"}
+                 {"value" "text/plain",
+                  "uuid" "fea4e0a7-d794-481d-9915-52f1be226714"}
+                 {"value" "image/jpeg",
+                  "uuid" "3f697f52-6a1c-4e2c-bd4b-13aaaf45f2e6"}
+                 {"value" "application/gzip",
+                  "uuid" "a8ee535a-8bc8-46fd-8b97-917bd7ea7666"}
+                 {"value" "application/x-hdf5",
+                  "uuid" "4e80047b-c50b-4805-ac68-789dbc38803f"}]}
```

Comment:
Why do we need to change the granule-data-format and mime-type here?  Is it that the kms structure changed for these two?

Reply:
Because what they were passing back wasn't correct.  We don't need to pass back a uuid structure that contains a uuid value, just uuid itself is what is needed.